pycc/lccwfn.py

```diff
@@ -0,0 +1,476 @@
+import time 
+#from timer import Timer
+import numpy as np
+from opt_einsum import contract  
+
+
+class lccwfn(object):
+    """
+    A PNO-CC object. 
+
+    Attributes
+    ----------
+    o : Numpy slice
+        occupied orbital subspace 
+    v : Numpy slice
+        virtual orbital subspace
+    no: int
+        number of (active) occupied orbitals
+    nv: int 
+        number of virtual orbitals
+    H: PyCC Hamiltonian object
+       Hamiltonian object with integrals, Fock matrices, and orbital coefficients
+    Local: PyCC Local object 
+       Local object with transformation matrices, local Fock matrices and integrals, etc. 
+      
+    Parameters
+    ----------
+    local: string 
+           type of local calculation ("PAO", "PNO", etc.)
```

Comment:
Is anything except "PNO" allowed?

Reply:
All the different local schemes can be used. I should add test cases for the others as well.